requirements.txt

```diff
@@ -4,3 +4,4 @@ pymatgen>=2018.11.30
 ase>=3.16.3b1
 mdf_toolbox>=0.4.2
 python-magic>=0.4.15
+git+https://github.com/CitrineInformatics/pif-dft.git@develop#egg=dfttopif
```

Comment:
`dfttopif` is on PyPI, so I think we should use that release instead of the GitHub repo (https://pypi.org/project/dfttopif/).

Reply:
For some reason, I thought the version of PyPi did not have the `files_to_pif` operation. Looking again, I'm not sure how I got to that conclusion.